app/views/publishers/home.html.slim

```diff
@@ -102,63 +102,11 @@ script type="text/html" id="confirm_default_currency_modal_wrapper"
-                = uphold_status_description(@publisher).html_safe
-              = link_to(uphold_authorization_description(@publisher),
-                        uphold_dashboard_url,
-                        id: "go_to_uphold", class: "btn btn-primary", :"data-piwik-action" => "GoToUpholdHomeClicked", :"data-piwik-name" => "Clicked", :"data-piwik-value" => "Dashboard", style: "display:none")
-              = link_to(uphold_authorization_description(@publisher),
-                        connect_uphold_publishers_path,
-                        id: "reconnect_to_uphold", class: "btn btn-primary", method: :patch, :"data-test" => "reconnect-button" , :"data-piwik-action" => "UpholdReconnectClicked", :"data-piwik-name" => "Clicked", :"data-piwik-value" => "Dashboard")
-              .reconnect-instructions
-                = "Having trouble with Uphold registration?"
-                br
-                = link_to(t(".uphold.connect_to_a_new_account"), connect_uphold_publishers_path,
-                      :"data-piwik-action" => "ConnectNewUpholdAccountClicked", method: :patch,
-                      :"data-piwik-name" => "Clicked", :"data-piwik-value" => "Dashboard")
-            .dashboard-panel--content#uphold_dashboard
-              .deposit-currency.field
-                label= t ".uphold.deposit_currency_label"
-                .value#default_currency_code= @publisher.uphold_connection.default_currency.present? ? @publisher.uphold_connection.default_currency : t(".uphold.no_currency_selected")
-                .action
-                  span= "("
-                  a#change_default_currency href="#"
-                    = t ".uphold.change"
-                  span= ")"
-              .last-deposit-date.field
-                label= t ".uphold.last_deposit_date"
-                .value.deposit-date#last_deposit_date=publisher_last_settlement_date(@publisher)
-            .dashboard-panel--content#last_settlement class=last_settlement_class(@publisher)
-              .last-deposit
-                = t ".uphold.last_deposit"
-                .amounts
-                  .bat
-                    - last_deposit_balance = publisher_last_settlement_bat_balance(@publisher)
-                    span.bat-amount#last_deposit_bat_amount= last_deposit_balance
-                    - unless last_deposit_balance == I18n.t("helpers.publisher.no_deposit")
-                      span.bat-code= " BAT"
-                  .converted#last_deposit_converted_amount
-                    = publisher_converted_last_settlement_balance(@publisher)
-                = link_to(t(".uphold.check_balance"), uphold_dashboard_url, class: "btn btn-primary",
-                        id: "uphold_check_balance", :"data-piwik-action" => "CheckUpholdBalanceClicked",
-                        :"data-piwik-name" => "Clicked", :"data-piwik-value" => "Dashboard")
-
+      .col-md.mb-4
+        - if !(@publisher.excluded_from_payout? || @publisher.paypal_locale?(params[:locale]))
```

Comment:
This logic confused me at first - I thought If a publisher is excluded from payout they will be shown the PayPal information. Can we nest the `if` statement so it's easier to read?

```slim
- unless @publisher.excluded_from_payout?
  - if @publisher.paypal_locale?(params[:locale]
     = render partial: "paypal_account_panel"
  - else 
    = render partial: "uphold_wallet_panel"
```